### PR TITLE
MNT Use GitHub Actions CI

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/gha-ci/issues/11

CI run - https://github.com/creative-commoners/silverstripe-installer/actions?query=branch%3Apulls%2F4.10%2Fmodule-standards

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
         "silverstripe/login-forms": "4.6.x-dev"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.5"
+        "phpunit/phpunit": "^9.5",
+        "mikey179/vfsstream": "^1.6.10"
```
Comment (GuySartorelli) [inline composer.json:14]: This is already defined in gha-ci.yml as being included for _all_ runs.
We should either explicitly require it in composer.json for every repo that needs it and remove from gha-ci.yml, or else just accept that it's included in ci and not bother adding here.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+jobs:
+  ci:
+    name: CI
+    # Only run cron on the silverstripe account
+    if: (github.event_name == 'schedule' && startsWith(github.repository, 'silverstripe/')) || (github.event_name != 'schedule')
+    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
+    with:
+      # installer contains a sample behat.yml file, though there are no behat tests to run
+      endtoend: false
```
Comment (GuySartorelli) [inline .github/workflows/ci.yml:19]: travis config explicitly defines some behat tests which are not being run here

File: composer.json [hunk 3]
```diff
@@ -10,7 +10,8 @@
         "silverstripe/login-forms": "4.6.x-dev"
     },
     "require-dev": {
```
Comment (GuySartorelli) [inline composer.json:12]: frameworktest and recipe-testing are defined in the travis config - are they needed here?

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (emteknetnz) [reply composer.json:14]: Technically it should be in composer, say if you're actually running unit tests on your local.  There's an existing card https://github.com/silverstripe/gha-ci/issues/6 to add it to composer.json

File: .github/workflows/ci.yml (hunk 2 above)
Comment (emteknetnz) [reply .github/workflows/ci.yml:19]: updated

File: composer.json (hunk 3 above)
Comment (emteknetnz) [reply composer.json:12]: yup, updated

**Review by GuySartorelli — APPROVED** [review]

**Pull request merged**